ibm/service/atracker/resource_ibm_atracker_target.go

```diff
@@ -102,6 +102,37 @@ func ResourceIBMAtrackerTarget() *schema.Resource {
 					},
 				},
 			},
+			"eventstreams_endpoint": &schema.Schema{
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Description: "Property values for an Event Streams Endpoint in requests.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"target_crn": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The CRN of the Event Streams instance.",
+						},
+						"brokers": &schema.Schema{
+							Type:        schema.TypeList,
+							Required:    true,
+							Description: "List of broker endpoints.",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+						"topic": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The messsage hub topic defined in the Event Streams instance.",
+						},
+						"password": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The user password (api key) for the message hub topic in the Event Streams instance.",
```

Comment:
Add sensitive : true

Reply:
password is senstive varaible so add sensitive : true
```
"password": &schema.Schema{
							Type:        schema.TypeString,
							Required:    true,
							Sensitive: true,
							Description: "The user password (api key) for the message hub topic in the Event Streams instance.",
						},
						```

Reply:
https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/sensitive-state#using-the-sensitive-flag